ecmd-core/capi/ecmdDataBufferBase.C

```diff
@@ -147,6 +147,14 @@ uint32_t fast_reverse32(uint32_t data) {
     fast_reverse8(data & 0x000000FF) << 24;
 }
 
+inline /* leave this inlined */
+uint32_t fast_reverse_bytes(uint32_t data) {
+    return (data & 0xFF000000) >> 24 |
+    (data & 0x00FF0000) >> 16 << 8 |
```

Comment:
This double shift doesn't seem necessary and is complicating things here?     fast_reverse32 needs to double shift because its taking data from a function call.  Here we are not and could simply be
>> 24
>> 8
<< 8
<< 24

Reply:
Thanks...fixed in latest revision.